Exercises-2-Basic/src/exercises/Num1.java

```diff
@@ -0,0 +1,46 @@
+package exercises;
+
+/* 1.
+Write a Java program to print the sum (addition), multiply, subtract, divide and 
+remainder of two numbers.
+Test Data:
+Input first number: 125
+Input second number: 24
+Expected Output :
+125 + 24 = 149
+125 - 24 = 101
+125 x 24 = 3000
+125 / 24 = 5
+125 mod 24 = 5*/
+
+public class Num1 {
+
+    public static void main(String[] args) {
+
+	int firstnumber = 200;
+	int secondnumber = 50;
+	int result;
+
+	// +
+	result = firstnumber + secondnumber;
+	System.out.println(firstnumber + " + " + secondnumber + " = " + result);
+
+	// -
+	result = firstnumber - secondnumber;
+	System.out.println(firstnumber + " - " + secondnumber + " = " + result);
+
+	// *
+	result = firstnumber * secondnumber;
+	System.out.println(firstnumber + " * " + secondnumber + " = " + result);
+
+	// /
+	result = firstnumber / secondnumber;
```

Comment:
what happens if `secondnumber` is 0?

Reply:
if secondnumber is 0 then the program will not run, so the secondnumber need to be bigger than 0,
added note.

Reply:
Actually, the program will fail, which is not something we want our users to experience, right?
We don't want them to see ugly errors.
So we need to make it better.
You wrote the comment "Important note: The second number must be bigger than 0", but don't forget that the users don't see the code and the comments, only developers do.

If the user enters an incorrect number, we need to tell them that.

For example,
```java
System.out.print("Input second number: ");
	int secondnumber = in.nextInt();
    if (secondnumber == 0) {
        System.out.print("The second must not be 0. Please execute the application again and input a correct number.");
        return;
    }
```

But in this case, the application exited and the user needs to execute it again.
Try to find a solution that allows the user to enter a new number right away.

Reply:
Good idea! I added and done.

Reply:
Try to find a solution that allows the user to enter a new number right away.

Reply:
still find solution...

Reply:
Can you give me a hint?

Reply:
What about making a loop to get the second number from the user?
Can you understand what is happening below?
Try to run it and enter the second number as 0.

```java
System.out.print("Input second number: ");
    int secondNumber = in.nextInt();
    for (int i = 0; i < 2; i++) {
        if (secondNumber == 0) {
            System.out.print("The second number must not be 0. Please enter the number again:");
            secondNumber = in.nextInt();
        }
        else {
            break;
        }
    }

    if (secondNumber == 0) {
        System.out.print("Exiting the application.");
        return;
    }
```

Reply:
done